client/public/index.html

```diff
@@ -5,30 +5,39 @@
 
 	  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 	  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-
+	  
 	  <title>CollaBoard</title>
-
-	  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+	  
+	  <!-- Fonts -->
+	  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+	  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	  
+	  <!-- Vendor Stylesheets -->
+	  <link href="./assets/css/vendor/materialize.min.css" rel="stylesheet"></link>
+ 	  <link href="./assets/css/vendor/prism.css" rel="stylesheet"/>
+ 	  <link href="./assets/css/vendor/editor.css" rel="stylesheet"/>
+	  
+ 	  <!-- App Stylesheets -->
 	  <link href="./assets/css/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+	  <link href="./assets/css/nav-bar.css" rel="stylesheet"></link>
+	  <link href="./assets/css/splash-container.css" rel="stylesheet"></link>
+	  <link href="./assets/css/whiteboard.css" rel="stylesheet"></link>
+	  <link href="./assets/css/features.css" rel="stylesheet"></link>
+	  <link href="./assets/css/product-select.css" rel="stylesheet"></link>	  
 	  <link href="./assets/css/styles/app.css" rel="stylesheet"/>
-	  <link href="./assets/css/nav-bar.css" rel="stylesheet"/>
-	  <link href="./assets/css/splash-container.css" rel="stylesheet"/>
-	  <link href="./assets/css/features.css" rel="stylesheet"/>
-	  <link href="./assets/css/product-select.css" rel="stylesheet"/>
-	  <link href="./assets/css/vendor/materialize.min.css" rel="stylesheet"/>
-	  <link href="./assets/css/vendor/prism.css" rel="stylesheet"/>
-	  <link href="./assets/css/vendor/editor.css" rel="stylesheet"/>
-
-	  <script src="js/whiteboard.js"></script>
 
 	</head>
 
 	<body>
-
+		
 		<div id="app"></div>
 
+		<!-- Vendor JS -->
 	  <script src="./assets/js/vendor/jquery-3.1.1.min.js"></script>
 	  <script src="./assets/js/vendor/materialize.min.js"></script>
+	  
+	  <!-- App JS -->
+	  <script src="js/whiteboard.js"></script>
```

Comment:
Let's not include the whiteboard file anymore. This is being handled within the react component now